implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/DynatraceConfig.java

```diff
@@ -165,6 +165,34 @@ default boolean exportMeterMetadata() {
         return getBoolean(this, "exportMeterMetadata").orElse(true);
     }
 
+    /**
+     * Intercept WARN messages generated by the Dynatrace registry and log them at INFO
+     * instead. This will reduce the WARN log volume generated by telemetry data exports
+     * in order to not hide business logic output.
+     * @return true if the logs should be intercepted and logged at INFO instead, false
+     * otherwise.
+     */
+    default boolean logWarningsAtInfo() {
+        if (apiVersion() == V1) {
+            return false;
+        }
+        return getBoolean(this, "logWarningsAtInfo").orElse(true);
+    }
+
+    /**
+     * Intercept ERROR messages generated by the Dynatrace registry and log them at INFO
+     * instead. This will reduce the ERROR log volume generated by telemetry data exports
+     * in order to not hide business logic output.
+     * @return true if the logs should be intercepted and logged at INFO instead, false
+     * otherwise.
+     */
+    default boolean logErrorsAtInfo() {
+        if (apiVersion() == V1) {
+            return false;
+        }
+        return getBoolean(this, "logErrorsAtInfo").orElse(true);
```

Comment:
```suggestion
        return getBoolean(this, "logWarningsAtInfo").orElse(false);
    }

    /**
     * Intercept ERROR messages generated by the Dynatrace registry and log them at INFO
     * instead. This will reduce the ERROR log volume generated by telemetry data exports
     * in order to not hide business logic output.
     * @return true if the logs should be intercepted and logged at INFO instead, false
     * otherwise.
     */
    default boolean logErrorsAtInfo() {
        if (apiVersion() == V1) {
            return false;
        }
        return getBoolean(this, "logErrorsAtInfo").orElse(false);
```
Shouldn't we keep the more verbose and thus safer default of `false` here and rather have users opt-out of adequate warnings if they consider them too noisy?

Reply:
Hm. I removed the WarnThenDebug loggers as well, so if we go to to verbose by default the number of messages might actually increase, but I guess you are right.

Reply:
Do you think it would be worth keeping both, or would we be overdoing it this way?

- logWarningsAtInfo=false: warnThenDebug
- logWarningsAtInfo=true:  infoThenDebug or just info

Reply:
It seems to get pretty complicated that way. I wonder if that isn't more trouble than it's worth, but I might give it a try and see where it ends up

Reply:
Updated it now, but I think it's kind of weird that a `WarnThenDebugLogger` doesn't log at Warn at all. It also required a change to the Micrometer upstream code, not sure if that change would be accepted.